File: java/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        for(int i = 0; i < my_string.length(); i++){
            char ch = my_string.charAt(i);
            if(ch >= 'a' && ch <= 'z')
                answer[ch - 'a' + 26]++;
            else
                answer[ch - 'A']++;
        }
        return answer;
    }
}

File: javascript/문자 개수 세기.js
const solution = (my_string) => {
    let answer = [];
    let alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
    let arr = [];
    arr.length = 52;
    arr.fill(0);

    my_string.split("").map((n) => {
        arr[alphabet.indexOf(n)] += 1
    })

    return arr;
}